airbyte-integrations/connectors/destination-postgres-strict-encrypt/src/test/resources/expected_spec.json

```diff
@@ -54,11 +54,145 @@
         "airbyte_secret": true,
         "order": 5
       },
+      "ssl_mode": {
+        "title": "SSL modes",
+        "description": "SSL connection configuration modes. \n <b>disable</b> - don't care about security and don't want to pay the overhead for encryption\n <b>allow</b> - don't care about security but will pay the overhead for encryption if the server insists on it\n <b>prefer</b> - don't care about encryption but will pay the overhead of encryption if the server supports it\n <b>require</b> - want my data to be encrypted, and I accept the overhead. I trust that the network will make sure I always connect to the server I want\n  <b>verify-ca</b> - want my data encrypted, and I accept the overhead. I want to be sure that I connect to a server that I trust\n  <b>verify-full</b> - want my data encrypted, and I accept the overhead. I want to be sure that I connect to a server I trust, and that it's the one I specify\n See more information - <a href=\"https://jdbc.postgresql.org/documentation/head/ssl-client.html\"> in the docs</a>.",
```

Comment:
Looks like these descriptions are copied from Postgres docs, which is OK, but I prefer to not copy/paste from external sources and I'd like us to more user friendly descriptions:
- change "SSL connection configuration modes" to "SSL connection modes"
- change description of "disable" to "Chose this mode to disable encryption of communication between Airbyte and source database"
- change description of "allow" to "Chose this mode to enable encryption *only* when required by the source database"
- change description of "prefer" to "Chose this mode to allow unencrypted connection *only* if the source database does not support encryption"
- change description of "require" to "Chose this mode to always require encryption. If the source database server does not support encryption, connection will fail"
- change description of "verify-ca" to "Chose this mode to always require encryption and to verify that the source database server has a valid SSL certificate"
- change description of "verify-full" to "This is the most secure mode. Chose this mode to always require encryption and to verify the identity of the source database server"